src/components/Input/Input.tsx

```diff
@@ -89,8 +90,15 @@ export default function Input(props: InputProps) {
     }
 
     let transformedInput = (
-        <input id={`input-${id}`} {...inputProps} placeholder={placeholder} />
+        <input
+            id={`input-${id}`}
+            {...inputProps}
+            placeholder={placeholder}
+            ref={ref}
```

Comment:
With attributes added into inputProps, such that anyone can pass any extra prop and have it attached to `<input>`, could you technically go 

<DS.Input attributes={{ref: newRef}} /> and should do that instead, 

Or is it still something we want to do to pass refs into all the form components?  (I think ReactRouter allows explicit passing of refs, so I'm up for either).

Reply:
It makes sense and it is technically possible to do the above, but `ref` isn't an HTML input attribute and is mostly used for React. So I would like to keep that separate. I think it also makes sense to keep it separate since it's using forwardRef for this purpose.